src/seedu/addressbook/data/person/Block.java

```diff
@@ -0,0 +1,14 @@
+package seedu.addressbook.data.person;
+
+public class Block {
+
+	private final String block;
+	
+	public Block(String blockNumber) {
```

Comment:
Also, you can consider the validation of block value.
e.g. Block number must be a number.